java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -972,4 +972,28 @@ public void zadd_illegal_arguments(BaseClient client) {
                         .build();
         options.toArgs();
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zrem(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        Map<String, Double> membersScores = Map.of("one", 1.0, "two", 2.0, "three", 3.0);
+        assertEquals(3, client.zadd(key, membersScores).get());
+        assertEquals(1, client.zrem(key, new String[] {"one"}).get());
+        assertEquals(2, client.zrem(key, new String[] {"one", "two", "three"}).get());
+        assertEquals(0, client.zrem("non_existing_set", new String[] {"member"}).get());
+    }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zcard(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        Map<String, Double> membersScores = Map.of("one", 1.0, "two", 2.0, "three", 3.0);
+        assertEquals(3, client.zadd(key, membersScores).get());
+        assertEquals(3, client.zcard(key).get());
+        assertEquals(1, client.zrem(key, new String[] {"one"}).get());
+        assertEquals(2, client.zcard(key).get());
```

Comment:
1. add a test to check the card of a non-set key. 
2. 
1. add a test to check the card of a non-existing key.
```
        assertEquals(0, client.zcard("non_existing_set").get());
```